### ISSUE TITLE
[Feature Request] Export PDF

### ISSUE THREAD

**Issue by ikteru** [excalidraw/excalidraw#1800]
### Feature:

At the moment, when we export something, it's usually in the form of an image or an SVG. What I'm proposing is an extra option of exporting it to PDF. 

### Suggestion:

The transformation from SVG to PDF isn't that complicated, this feature shouldn't cause too much overhead to implement if you just transform the SVG into PDF format.

**Comment by Frischid** [comment]
As in #1806 discussed the huge jspdf library is nice but it seems to be too big.
I read a little bit on this wiki: http://www.p2501.ch/pdf-howto/grundlagen/start (sadly it's old and written in German but it's very detailed).
It seems to be not that hard to use parts from svg export and export the paths in the correct way to a pdf file without using jspdf.
Exporting pdf this way would also make sure we still have paths instead of an image (jspdf seems to generate images out of svg).
@ikteru do you think pdf export is still a nice feature to be implemented rather than using "Print to PDF"?

**Comment by linguax-gc** [comment]
I add my vote for this feature. Most times I end up having to create a PDF, and it's not that easy. Also, when exporting images and converting them to PDF with Acrobat or whatever, links are lost, and I have to add them manually with another app. So this would be extremely helpful for me.

**Comment by eric-schleicher** [comment]
Yup coming back to this again.   this is still a very important feature.  being able to send someone a PDF with working link is the specific need.  exporting PNGs and PDF sin them doesn't cut it.   

Is there a lite implementation of this where the PDF is raster with  imagemap type links?

**Comment by dacasine** [comment]
Poor PDF export is something extremely problematic in business cases. I’d like to share my drawings and include them in books / articles / etc. but SVG is not as accepted as the developers seems to think. I think that this lack of feature is quite sad for the future of this project.

**Comment by eric-schleicher** [comment]
I understand the frustration, but I'm not so doom and gloom-y about it.   one big lesson about open source is that the nexus of need and ability is unpredictable, but generally inevitable if it's actually useful.   IMHO excalidraw is the future of open source whiteboarding, and so much good comes from all in on SVG as a WC3 standard.  It's worth having to wait to pick up these types of capabilities.

**Comment by Mrazator** [comment]
Short update after some time, our premium [plus.excalidraw.com](https://plus.excalidraw.com/) has PDF export support, rendering to PDF server-side. For the [excalidraw.com](https://excalidraw.com/), we will likely need to find some client-side alternative in the future due to its E2EE/"serverless" nature.

**Comment by mitar** [comment]
PRs are still open? https://github.com/excalidraw/excalidraw/pull/10854
